sdk/python/kfp/compiler/pipeline_spec_builder.py

```diff
@@ -2063,11 +2063,21 @@ def write_pipeline_spec_to_file(
 
 def _merge_pipeline_config(pipelineConfig: pipeline_config.PipelineConfig,
                            platformSpec: pipeline_spec_pb2.PlatformSpec):
-    # TODO: add pipeline config options (ttl, semaphore, etc.) to the dict
-    # json_format.ParseDict(
-    #     {'pipelineConfig': {
-    #         '<some pipeline config option>': pipelineConfig.<get that value>,
-    #     }}, platformSpec.platforms['kubernetes'])
+    workspace = pipelineConfig.workspace
+    if workspace is None:
+        return platformSpec
+
+    workspace_dict = {
+        'size': workspace.size,
+    }
+
+    if workspace.kubernetes:
+        workspace_dict['kubernetes'] = {
+            'pvcSpecPatch': workspace.kubernetes.pvc_spec_patch
+        }
```

Comment:
I wonder if we should move this patch dict construction into the WorkspaceConfig class itself, such that we would simply call something like `workspace_dict = get_workspace_spec()` which returns this dict.  It would consolidate construction/formatting into the class itself in case we want to expand it later.

That said, this is splitting hairs so strictly optional/not worth blocking PR on

Reply:
I think that's a good idea. What do you think @VaniHaripriya ?

Reply:
That's a good idea @gmfrasca . I will update the  WorkspaceConfig class.